crawler/utils/dockerutils.py

```diff
@@ -360,3 +363,31 @@ def get_docker_container_rootfs_path(long_id, inspect=None):
         raise DockerutilsException('Not supported docker storage driver.')
 
     return rootfs_path
+
+
+def poll_container_create_events(timeout=0.1):
+    try:
+        client = docker.Client(base_url='unix://var/run/docker.sock',
```

Comment:
I'm not sure as I haven't tested that API, and the docs are not clear about it. But at least this `since` parameter makes me think that either the daemon or the client have to store the events (http://docker-py.readthedocs.io/en/stable/client.html?highlight=events):
```
events()
Get real-time events from the server. Similar to the docker events command.

Parameters:	
since (UTC datetime or int) – Get events from this point
until (UTC datetime or int) – Get events until this point
```